docs/user/appendix/changelog/index.md

```diff
@@ -1,6 +1,4 @@
 # Changelog
 
-This section contains the changelog for both Glean clients.
-
-- [The Glean SDK](./sdk.md)
-- [Glean.js](./js.md)
+- [`mozilla/glean` (glean-core)](./sdk.md)
```

Comment:
I'm not sure that's "glean-core". Isn't glean-core the Rust core alone? Isn't this just Glean, given that you're referring to the other SDK as Glean.js?

Reply:
Good question, not really sure here. Maybe I could remove this completely and just leave the repository name to avoid the rabbit hole. This is kinda old, but I was clinging to [this](https://mozilla.github.io/glean/book/appendix/changelog/sdk.html#v1900-2019-10-22) changelog entry that referred to the whole thing as "glean-core".

Reply:
> Isn't this just Glean

Isn't Glean the collection of SDKs + pipeline + tools?

Reply:
> 
> 
> > Isn't this just Glean
> 
> Isn't Glean the collection of SDKs + pipeline + tools?

I've been Dexter'd. You are correct. Then Glean.js is probably Glean JS SDK and Glean is "Other Glean SDKs"?

Reply:
So, there is also the horrible solution of calling them, "Rust core-based SDKs" and "JavaScript core-based SDKs". Which I started with and then removed. I did use it in the SDKs overview page I edited in this PR, see: https://github.com/mozilla/glean/pull/1859/files#diff-5358def7708fb975f1114844a9a38f2f5c0972f6262ec00962303e6029e1d605R18-R23 and https://github.com/mozilla/glean/pull/1859/files#diff-5358def7708fb975f1114844a9a38f2f5c0972f6262ec00962303e6029e1d605R84-R89.

Reply:
Talked to @badboy about this just now and leaving just the repository name seems like a reasonable solution here.